tests/unit/constraints/test_tabular.py

```diff
@@ -990,7 +990,7 @@ def test__validate_metadata_specific_to_constraint_datetime_error(self):
 
         # Run
         error_message = re.escape(
-            'An Inequality constraint is being applied to mismatched sdtypes '
+            'An Inequality constraint is being applied to mismatched sdtype columns '
```

Comment:
Unrelated. Phrasing was confusing, it made it seem the values were sdtypes when they are actually columns.

Reply:
minor: can we reword to `is being applied to columns with mismatched sdtypes` then? I think that makes more sense